helm/prometheus-rules/templates/kaas/phoenix/alerting-rules/aws-cloud-components.rules.yml

```diff
@@ -0,0 +1,32 @@
+{{- if eq .Values.managementCluster.provider.kind "capa" }}
+# This rule applies to capa management clusters only
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: aws-cloud-components.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: aws-cloud-components
+    rules:
+    - alert: FluxHelmReleaseFailed
+      annotations:
+        description: |-
+          {{`Flux HelmRelease {{ $labels.name }} in ns {{ $labels.exported_namespace }} on {{ $labels.installation }}/{{ $labels.cluster_id }} is stuck in Failed state.`}}
+        opsrecipe: fluxcd-failing-helmrelease/
+      expr: gotk_reconcile_condition{type="Ready", status="False", kind="HelmRelease", cluster_type="management_cluster", exported_namespace!="flux-giantswarm", name=~".*(aws-ebs-csi-driver|cloud-provider-aws)"} > 0
```

Comment:
Just one question here it looks like `gotk_reconcile_condition` is deprecated and Flux team is suggesting to use `gotk_resource_info`, do we wanna switch here?

Reply:
I don't think we have that metric

![image](https://github.com/user-attachments/assets/d4084f25-bb7a-4577-a0c6-271afa6b9903)

Reply:
Ok thanks 👍🏻, so maybe we don't run the min. required version for flux 🤔